regions/io/read_ds9.py

```diff
@@ -192,20 +194,33 @@ def ds9_region_list_to_objects(region_list):
             else:
                 raise ValueError("No central coordinate")
         else:
+            # Note: this should effectively never happen, because it would
+            # imply that the line_parser found a region that didn't match the
+            # above region types.  However, this can help with development,
+            # since we could in theory implement more region types in the line
+            # parser and forget to add them here.
+            warn("Skipping region with coords {0} because its type '{1}'"
+                 " is not recognized."
+                 .format(str(coord_list), region_type),
+                 AstropyUserWarning
+                )
             continue
-        reg.vizmeta = {key: meta[key] for key in meta.keys() if key in viz_keywords}
+        reg.visual = {key: meta[key] for key in meta.keys() if key in viz_keywords}
         reg.meta = {key: meta[key] for key in meta.keys() if key not in viz_keywords}
         output_list.append(reg)
     return output_list
 
 
-def ds9_string_to_objects(region_string):
+def ds9_string_to_objects(region_string, warn_skipped=False):
     """Parse ds9 region string to region objects
 
     Parameters
     ----------
     region_string : str
         DS9 region string
+    warn_skipped : bool
+        Print a warning if there is a skipped (commented) line?
+        Can set to ``False`` or ``'raise'`` if you want an exception instead.
```

Comment:
Like I said in the other comment, I'd prefer to call it "error".
And as-is, the option is documented as `bool`, but you also accept a string `raise`.
That's no good.
Let's make it a multi-choice string option.